src/ai/backend/runner/entrypoint.sh

```diff
@@ -55,7 +55,7 @@ if [ $USER_ID -eq 0 ]; then
     /opt/backend.ai/bin/python -s /opt/kernel/fantompass.py > "$HOME/.password"
     export ALPHA_NUMERIC_VAL=$(cat $HOME/.password)
     chmod 0644 "$HOME/.password"
-    echo "work:$ALPHA_NUMERIC_VAL" | chpasswd
+    echo "work:$ALPHA_NUMERIC_VAL" | chpasswd -c SHA512
```

Comment:
In the case where the issue occurred, there shouldn’t be a problem since there were no previously stored hash values, right?
In the normal operation scenario, hash values would have been stored, but changing the hashing algorithm shouldn’t cause any issues?

Reply:
I can find that Ubuntu introduced SHA512 password hashing algorithm at [8.10](https://jonmccune.wordpress.com/2009/07/28/1-vs-6-in-etcshadow/) and RHEL from [5](https://access.redhat.com/solutions/221403). So yes, at least for the Linux distros supported by Backend.AI, it will be fine to fix the password hashing method.